share/workspaces/Basic.xml

```diff
@@ -585,31 +585,31 @@
         <mag>0.65</mag>
         <grid>1</grid>
         <moreElements>1</moreElements>
-        <Cell name="Tempo text">
+        <Cell name="Half note = 80 BPM">
           <mag>1.5</mag>
           <Tempo>
             <tempo>2.66667</tempo>
             <followText>1</followText>
             <text><sym>metNoteHalfUp</sym> = 80</text>
             </Tempo>
           </Cell>
-        <Cell name="Tempo text">
+        <Cell name="Quarter note = 80 BPM">
           <mag>1.5</mag>
           <Tempo>
             <tempo>1.33333</tempo>
             <followText>1</followText>
             <text><sym>metNoteQuarterUp</sym> = 80</text>
             </Tempo>
           </Cell>
-        <Cell name="Tempo text">
+        <Cell name="Eigth note = 80 BPM">
           <mag>1.5</mag>
           <Tempo>
             <tempo>0.666667</tempo>
             <followText>1</followText>
             <text><sym>metNote8thUp</sym> = 80</text>
             </Tempo>
           </Cell>
-        <Cell name="Tempo text">
+        <Cell name="Dotted Quarter note = 80 BPM">
```

Comment:
Should be "Dotted quarter note = 80 BPM". No need for Q to be capitalized.

Reply:
done.